pyaerocom/aeroval/coldatatojson_engine.py

```diff
@@ -149,7 +151,7 @@ def _prepare_regions_json_helper(region_ids):
 
         latr = reg.lat_range
         lonr = reg.lon_range
-        if any(x is None for x in (latr, lonr)):
+        if any(x is None for x in (latr, lonr)): # pragma: no cover
```

Comment:
```suggestion
        if any(x is None for x in (latr, lonr)): 
```
I think tests should cover this.